src/database/db_build.sql

```diff
@@ -0,0 +1,46 @@
+BEGIN;
+
+DROP TABLE IF EXISTS mentors, questions, singing cascade;
+
+CREATE TABLE mentors(
+  id        SERIAL    PRIMARY KEY,
+  name      TEXT      NOT NULL,
+  imageUrl  TEXT      NOT NULL
+);
+
+CREATE TABLE questions(
+  id          SERIAL    PRIMARY KEY,
+  category    TEXT      NOT NULL,
+  question    TEXT      NOT NULL
+);
+
+CREATE TABLE singing(
+  id        SERIAL        PRIMARY KEY,
+  name      VARCHAR(20)   NOT NULL,
```

Comment:
I think we should join the singing table to the mentors' table using the id column  and there is no need for the name